src/core/components/ButtonIcon.tsx

```diff
@@ -49,10 +50,20 @@ const ButtonIcon = ({
     : 'top';
   const tooltipDelay = isTooltipProps(tooltip) ? tooltip.delay : undefined;
 
-  const tooltipStyle = tooltipDelay
-    ? { transitionDelay: `${tooltipDelay}ms` }
+  const backgroundColorOveride = isTooltipProps(tooltip)
+    ? tooltip.backgroundColor
     : undefined;
 
+  const tooltipStyle = tooltipDelay
+    ? {
+        transitionDelay: `${tooltipDelay}ms`,
+      }
+    : backgroundColorOveride
+      ? {
+          backgroundColor: `${backgroundColorOveride}`,
+        }
```

Comment:
We may need to set a custom background to make the tooltip contrast with the element behind it.